src/ThingooConnector.cpp

```diff
@@ -0,0 +1,65 @@
+/***************************************************************
+   ThingooConnector is a library for the ESP8266/Arduino platform
+   to communicate with Thingoo API
+   Built for KNI Project "Thingoo" https://github.com/ThingooKNI/
+ **************************************************************/
+#include "ThingooConnector.h"
+
+ThingooConnector::ThingooConnector(const char *host)
+{
+	_host = String(host);
+	_host.replace("/", "");
```

Comment:
This would replace **all** slashes, not only the trailing one. And we only want to delete the trailing one :)